exporter/elasticsearchexporter/README.md

```diff
@@ -213,3 +213,89 @@ Other metric types (Histogram, Exponential Histogram, Summary) are ignored.
 [data stream]: https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html
 [ecs]: https://www.elastic.co/guide/en/ecs/current/index.html
 [SemConv]: https://github.com/open-telemetry/semantic-conventions
+
+
+## ECS Mapping
+
+When `mode` is set to `ecs`, `elasticsearchexporter` performs conversions for resource-level attributes from their Semantic Conventions (SemConv) names to equivalent Elastic Common Schema (ECS) names.
+
+If the target ECS field name is specified as an empty string (""), the converter will neither convert the SemConv key to the equivalent ECS name nor pass through the SemConv key as-is to become the ECS name.
+
+"Preserved" means the convention should be preserved in ECS mode. This can happen when an attribute needs to be mapped to an ECS equivalent but also preserved in its original form.
+
+| Semantic Convention Name | ECS Name | Preserve |
+|--------------------------|----------|----------|
+| cloud.platform           | cloud.service.name | false |
+| container.image.tags     | container.image.tag | false |
+| deployment.environment   | service.environment | false |
+| host.arch                | host.architecture | false |
+| host.name                | host.hostname | true |
+| k8s.deployment.name      | kubernetes.deployment.name | false |
+| k8s.namespace.name       | kubernetes.namespace | false |
+| k8s.node.name            | kubernetes.node.name | false |
+| k8s.pod.name             | kubernetes.pod.name | false |
+| k8s.pod.uid              | kubernetes.pod.uid | false |
+| os.description           | host.os.full | false |
+| os.name                  | host.os.name | false |
+| os.type                  | host.os.platform | false |
+| os.version               | host.os.version | false |
+| process.executable.path  | process.executable | false |
+| process.runtime.name     | service.runtime.name | false |
+| process.runtime.version  | service.runtime.version | false |
+| service.instance.id      | service.node.name | false |
+| telemetry.distro.name    | "" | false |
+| telemetry.distro.version | "" | false |
+| telemetry.sdk.language   | "" | false |
+| telemetry.sdk.name       | "" | false |
+| telemetry.sdk.version    | "" | false |
+
+### Compound Mapping
+
+There are ECS fields that are not mapped easily 1 to 1 but require more advanced logic.
+
+#### `agent.name`
+
+The agent name takes the form of a compound name consisting of 3 components:
```

Comment:
Can those component values contain the character `/` ?

Reply:
i dont see why not, but having `telemetry.sdk.language/name/version` none of them has this so far

Reply:
It'd be great that it doesn't happen in the future too.